lib/ansible/modules/storage/zfs/zpool_facts.py

```diff
@@ -157,11 +160,15 @@ def get_facts(self):
         cmd = [self.module.get_bin_path('zpool')]
 
         cmd.append('get')
-        cmd.append('-H')
+        if not self.is_sunos:
+            # solaris 11 does not support -H
+            cmd.append('-H')
         if self.parsable:
             cmd.append('-p')
-        cmd.append('-o')
-        cmd.append('name,property,value')
+        if self.is_freebsd:
```

Comment:
i believe -o is supported on linux also, it relates openzfs more than a platform